src/Lumper.UI/ViewModels/Shared/Pakfile/PakfileLumpViewModel.cs

```diff
@@ -175,17 +188,42 @@ public void UpdatePathReferences(string newPath, string oldPath)
             var updatedNp = newPath;
             var matchIndex = -1;
             var changes = 0;
-            while ((matchIndex =
-                       entry.Content.IndexOf(
-                           opNoPrefix,
-                           startIndex: matchIndex + 1,
-                           StringComparison.Ordinal))
-                   != -1)
+
+            var tryWithoutExtension = IgnorableExtensions.TryGetValue(Path.GetExtension(entry.Key), out var opNoExtension);
+            while (true)
             {
+                var match = entry.Content.IndexOf(opNoPrefix, startIndex: matchIndex + 1, cmp);
```

Comment:
I agree. Currently the usage rules are defined in our editorconfig though, using 
```
csharp_style_var_for_built_in_types = true:warning
csharp_style_var_when_type_is_apparent = true:warning
csharp_style_var_elsewhere = false:warning
```
https://learn.microsoft.com/en-gb/dotnet/fundamentals/code-analysis/style-rules/ide0007-ide0008#csharp_style_var_for_built_in_types

As far as I can tell, `csharp_style_var_for_built_in_types` has precedence over `csharp_style_var_when_type_is_apparent`, so we've had to be using explicit types for *all* built-ins.

On balance I'd prefer the explicit types, worse to have ambiguous types like in this case, than to have some some cases that are unnecessarily explicit. And I have no issue with `int i = 0;` instead of `var i = 0;`.

I don't want to change all these now, however. The rebasing would kill me. As with CSharpier, I'd prefer we wait until this PR is complete, then decide final formatting rules / make any editorconfig changes, and do everything at once.